### PR TITLE
Queue | Search for other appeals

### PR DESCRIPTION
**Pull request by dannysellers**
Connects #4258 

## AC
- [ ] Verify that an appeal can be searched by Veteran ID
- [ ] When the user searches for an appeal by Veteran ID, the user selects the claims folder to open with the existing modal in Reader
- [ ] When the user selects an appeal from the modal, the user is redirected to the reader claims folder (`/reader/appeal/${vacols_id}/documents`)

### CODE REVIEW

**Comment by dannysellers** [comment]
queue_case_search

**Comment by amprokop** [comment]
image

@shellicious @sneha-pai — this is what the current error state looks like, can yall make a new ticket if there are any tweaks you want made to this?

File: client/app/components/SearchBar.jsx [hunk 1]
```diff
@@ -94,8 +94,7 @@ export default class SearchBar extends React.Component {
       submitUsingEnterKey,
       placeholder,
       internalText,
-      spinnerColor,
-      styling
```
Comment (amprokop) [inline client/app/components/SearchBar.jsx]: I assume you grepped around and ensured this wasn't used anywhere?

File: client/app/queue/QueueApp.jsx [hunk 2]
```diff
@@ -4,44 +4,43 @@ import { BrowserRouter } from 'react-router-dom';
 import _ from 'lodash';
 import { css } from 'glamor';
 
-import SearchBar from '../components/SearchBar';
+import CaseSelectSearch from '../reader/CaseSelectSearch';
```
Comment (amprokop) [inline client/app/queue/QueueApp.jsx]: Think we should put a TODO to move this component to queue once we remove this from Reader?

File: client/app/components/SearchBar.jsx (hunk 1 above)
Comment (dannysellers) [reply client/app/components/SearchBar.jsx]: Yeah, this was an attribute I had added in the first Queue table ticket (#4407)

**Pull request merged**